eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -135,23 +135,36 @@ instance DecCBOR AssetName where
 
 -- | Policy ID
 newtype PolicyID c = PolicyID {policyID :: ScriptHash c}
-  deriving (Show, Eq, EncCBOR, DecCBOR, Ord, NoThunks, NFData, Generic)
+  deriving
+    ( Show
+    , Eq
+    , Ord
+    , Generic
+    , NoThunks
+    , NFData
+    , EncCBOR
+    , DecCBOR
+    , ToJSON
+    , FromJSON
+    , ToJSONKey
+    , FromJSONKey
+    )
 
 -- | The MultiAssets map
 newtype MultiAsset c = MultiAsset (Map (PolicyID c) (Map AssetName Integer))
   deriving (Show, Generic, ToJSON)
 
 instance Crypto c => Eq (MultiAsset c) where
-  (MultiAsset x) == (MultiAsset y) = pointWise (pointWise (==)) x y
```

Comment:
haha almost sure these are my parentheses!  It's not possible to use them in a more useless way :laughing: